### PR TITLE
Some improvements on private registry credentials docs

### PR DESCRIPTION
**Pull request by ifbyol**
* Added a note in the dashboard section to indicate a couple of example of valid hostnames for registries
* Added a link in the dashboard section where we talk about private registry to the section where we explain how to configure them using CRDs
* Added example of valid hostnames in the section where we explain how to create CRDs for registries

### CODE REVIEW

**Review by mnevadom — APPROVED** [review]

File: src/content/administration/dashboard.mdx [hunk 1]
```diff
@@ -177,14 +177,18 @@ From this page you can follow a simple step-by-step guide to get the major compo
...
 
 If the setting [`daemonset.configurePrivateRegistriesInNodes.enabled`](/self-hosted/administration/configuration.mdx/#daemonset) is enabled, Okteto's daemonset will also configure these credentials in the nodes so the cluster can also pull images from those registries.
 
 For each configured registry, you will see a row with the type (Static or AWS) and the registry hostname. The 2 types of credentials supported are:
 - Static: Registry configuration using static username and password credentials (e.g. Docker Hub)
 - AWS: [ECR registry](https://docs.aws.amazon.com/ecr/index.html) configuration using Access Key and Access Secret. Okteto takes care of refreshing docker credentials every 4 hours using the key and secret configured. The hostname of the registry must end with `.amazonaws.com`.
 
+:::note
+Registries have to be included specifying a valid RFC1123 host name. For example, `index.docker.io` or `111122223333.dkr.ecr.eu-central-1.amazonaws.com`.
```
Comment (omnomagonz) [inline src/content/administration/dashboard.mdx:189]:
```suggestion
Registries must be included using a valid RFC1123 host name. For example, `index.docker.io` or `111122223333.dkr.ecr.eu-central-1.amazonaws.com`.
```

File: src/content/self-hosted/install/crds.mdx [hunk 2]
```diff
@@ -20,11 +20,11 @@ You can let Okteto install these CRDs or you can install them manually on your o
...
 
 Once the secret is created, you will need to create a resource with the apiVersion `admin.okteto.com/v1`, which is one of the CRDs installed by Okteto, referring to the secret which contains the credentials.
 
-In that way, if you want to define static credentials (username and password) for a private registry, you just need to create a secret in the Okteto namespace with the user and pass to be used. After that, you have to define a CRD with some information. The name of the resource has to be the hostname of the registry, so imagine you want to add a configuration for docker's registry, you will need to create a resource like this:
+In that way, if you want to define static credentials (username and password) for a private registry, you just need to create a secret in the Okteto namespace with the user and pass to be used. After that, you have to define a CRD with some information. The name of the resource has to be the hostname of the registry (e.g. `index.docker.io` or `111122223333.dkr.ecr.eu-central-1.amazonaws.com`), so imagine you want to add a configuration for docker's registry, you will need to create a resource like this:
```
Comment (omnomagonz) [inline src/content/self-hosted/install/crds.mdx:27]:
```suggestion
If you want to define static credentials (username and password) for a private registry you need to create a secret in the Okteto namespace with the username and password to be used. Once created, you need to define a CRD using the name of the resource as the hostname of the registry (e.g. `index.docker.io` or `111122223333.dkr.ecr.eu-central-1.amazonaws.com`).  For example, if you want to add a configuration for Docker's registry, you will need to create a resource like this:
```

**Review by omnomagonz — APPROVED** [review]
non-blocking suggestions

File: src/content/administration/dashboard.mdx [hunk 3]
```diff
@@ -177,14 +177,18 @@ From this page you can follow a simple step-by-step guide to get the major compo
 
 ## Registry Credentials
 
-In this section of the Admin dashboard you can manage the private registry credentials for your Okteto instance. The credentials you configure here are injected by Okteto into any operation (e.g. build, deploy) performed by developers so they don't need direct access to the credentials. Once you configure the credentials here all developers in your Okteto instance will be able to use the respective registries while developing with Okteto. 
+In this section of the Admin dashboard you can manage the private registry credentials for your Okteto instance. The credentials you configure here are injected by Okteto into any operation (e.g. build, deploy) performed by developers so they don't need direct access to the credentials. Once you configure the credentials here all developers in your Okteto instance will be able to use the respective registries while developing with Okteto. You can also manage these credentials directly in Kubernetes cluster using the [CRDs](/self-hosted/install/crds.mdx#private-registries) created by Okteto.
```
Comment (rberrelleza) [inline src/content/administration/dashboard.mdx:180]:
```suggestion
In this section of the Admin dashboard you can manage the private registry credentials for your Okteto instance. The credentials you configure here are injected by Okteto into any operation (e.g. build, deploy) performed by developers so they don't need direct access to the credentials. Once you configure the credentials here all developers in your Okteto instance will be able to use the respective registries while developing with Okteto. You can also manage these credentials directly in your Kubernetes cluster using the [CRDs](/self-hosted/install/crds.mdx#private-registries) created by Okteto.
```

**Pull request merged**